docs/config/key-tables.md

```diff
@@ -31,22 +31,26 @@ wezterm.on("update-right-status", function(window, pane)
 end)
 
 return {
-  leader = {key="Space", mods="CTRL|SHIFT"},
+  leader = { key = "Space", mods = "CTRL|SHIFT" },
   keys = {
     -- CTRL+SHIFT+Space, followed by 'r' will put us in resize-pane
     -- mode until we cancel that mode.
-    {key="r", mods="LEADER", action=act.ActivateKeyTable{
-      name="resize_pane",
-      one_shot=false,
-    }},
+    { key = "r", mods = "LEADER", action = act.ActivateKeyTable({
+      name = "resize_pane",
+      one_shot = false,
+    }) },
 
     -- CTRL+SHIFT+Space, followed by 'a' will put us in activate-pane
     -- mode until we press some other key or until 1 second (1000ms)
     -- of time elapses
-    {key="a", mods="LEADER", action=act.ActivateKeyTable{
-      name="activate_pane",
-      timeout_milliseconds=1000,
-    }},
+    {
+      key = "a",
+      mods = "LEADER",
+      action = act.ActivateKeyTable({
```

Comment:
and these `act` cases don't need parens. What are the chances that we can control the formatting to be this specific?